meta_row.py

```diff
@@ -15,24 +15,24 @@ class MetaRow:
     LineStart: int
     LineEnd: int
     GroundTruth: str
-    WithWords: str
+    
     ValueStart: int
     ValueEnd: int
-    InURL: str
-    InRuntimeParameter: str
-    CharacterSet: str
+    
+    
+    
     CryptographyKey: str
     PredefinedPattern: str
-    VariableNameType: str
-    Entropy: float
-    Length: int
-    Base64Encode: str
-    HexEncode: str
-    URLEncode: str
+    
+    
+    
+    
+    
+    
     Category: str
 
     def __init__(self, row: dict):
-        if not isinstance(row, dict) or self.__annotations__.keys() != row.keys():
+        if not isinstance(row, dict) :
```

Comment:
```suggestion
        if not isinstance(row, dict) or self.__annotations__.keys() != row.keys():
```
The check should be kept